### PR TITLE
flatted the css, added more classes to elements

### PR DESCRIPTION
**Pull request by ChCosmin**

### CODE REVIEW

File: this 2.0/assets/evaluations.less [hunk 1]
```diff
@@ -5,41 +5,42 @@
...
+.eval-table-head {
+    background-color: #03a9f4;
+    color: #fff;
+}
 
-    td:nth-child(4) {
-        text-align: right;
-        padding-right: 5px;
-    }
+.eval-table-col:nth-child(4) {
```
Comment (softvision-alexcarcea) [inline this 2.0/assets/evaluations.less]: Just the fourth? Generally speaking? Or are you targeting the last one? Would this ever be reused on columns other than the 4th?

File: this 2.0/assets/new_evaluations.less [hunk 2]
```diff
@@ -6,70 +6,73 @@
...
+.skill-list-item {
+    display: inline-block;
+    text-align: center;
+    flex: 1;
 
-        label {
-            background-color: #03a9f4;
-            color: #fff;
-        }
+    *:not(input) {
```
Comment (softvision-alexcarcea) [inline this 2.0/assets/new_evaluations.less]: What types of elements would you normally expect to hold in a `.skill-list-item` container?

File: this 2.0/assets/new_evaluations.less [hunk 3]
```diff
@@ -6,70 +6,73 @@
...
+.skill-list-radio {
+    margin: 10px;
+}
 
-        *:not(input) {
-            display: block;
-            padding: 10px;
-        }
-    }
+.skill-list-radio[type='radio'] {
```
Comment (softvision-alexcarcea) [inline this 2.0/assets/new_evaluations.less]: So, are there instances of `.skill-list-radio` components that don't have a `type` of `radio`? :confused:

File: this 2.0/new_evaluations.html [hunk 4]
```diff
@@ -8,157 +8,159 @@
...
-				</a>
-			</div>
-			<div class="menu-items col-2-menu">
-				<a href="evaluations.html">Evaluations</a>
-				<a class="active" href="#">New Evaluation</a>
-			</div>
-			<div class="menu-items logout col-1-menu">
-				<a href="login.html">Logout</a>
+	<div id="new_evaluations"></div>
+	<!-- <header>
```
Comment (softvision-alexcarcea) [inline this 2.0/new_evaluations.html]: If no longer needed, let's go ahead and remove these.

File: this 2.0/assets/evaluations.less (hunk 1 above)
Comment (ChCosmin) [inline this 2.0/assets/evaluations.less]: fixed, thought :last-child worked like :nth-child(first-child)

Comment (softvision-alexcarcea) [inline this 2.0/assets/evaluations.less]: It worked like what now? :|

File: this 2.0/assets/new_evaluations.less (hunk 2 above)
Comment (ChCosmin) [inline this 2.0/assets/new_evaluations.less]: changed * with the 2 tags's class names in the skill-list-item tag,, that being said i'll have to add a new class to the css everytime i add a new tag...soooo isn't it better to just tag all elements inside the skill-list-items (except the input tags)?

File: this 2.0/assets/new_evaluations.less (hunk 3 above)
Comment (ChCosmin) [inline this 2.0/assets/new_evaluations.less]: initially it was input[type='radio'] but then i gave it a class and didn't think i'd need to take out the type selector, thanks for pointing that out

File: this 2.0/new_evaluations.html (hunk 4 above)
Comment (ChCosmin) [inline this 2.0/new_evaluations.html]: i prefer to have the original html there until i know the js part is fully working, so i'll keep it a bit more

File: this 2.0/assets/new_evaluations.less (hunk 2 above)
Comment (softvision-alexcarcea) [inline this 2.0/assets/new_evaluations.less]: It is, if you're absolutely sure that all other tags will need to have this styling. So, are you absolutely, positively, irrevocably certain? :)

File: this 2.0/assets/evaluations.less (hunk 1 above)
Comment (ChCosmin) [inline this 2.0/assets/evaluations.less]: it worked with just .eval-table-col:last-child {}

File: this 2.0/assets/new_evaluations.less (hunk 2 above)
Comment (ChCosmin) [inline this 2.0/assets/new_evaluations.less]: yes

**Pull request merged**